java/arcs/core/storage/StorageProxy.kt

```diff
@@ -192,12 +192,14 @@ class StorageProxy<Data : CrdtData, Op : CrdtOperation, T>(
     }
 
     private suspend fun notifyUpdate(ops: List<Op>) = forEachHandle { handle ->
-        ops.forEach { handle.callback?.onUpdate(it) }
+        ops.forEach {
+            handle.onUpdate(it)
+        }
     }
 
-    private suspend fun notifySync() = forEachHandle { it.callback?.onSync() }
+    private suspend fun notifySync(versionMap: VersionMap) = forEachHandle { it.onSync(versionMap) }
```

Comment:
I think this needs to be a copy since multiple handles would otherwise get the same versionMap instance. onUpdate needs the same thing.

Reply:
Oh, whoops! Good catch.